src/stacked_borrows.rs

```diff
@@ -334,22 +341,21 @@ impl<'tcx> Stack {
 
     /// Test if a memory `access` using pointer tagged `tag` is granted.
     /// If yes, return the index of the item that granted it.
+    /// `range` refers the entire operation, and `offset` refers to the specific location in
+    /// `range` that we are currently checking.
```

Comment:
`offset` is relative to `alloc_id` though, not relative to `range`, right?

Reply:
Correct. My initial wording was ambiguous.